todo_app/data/models/item.py

```diff
@@ -0,0 +1,25 @@
+from enum import Enum
+from todo_app.data.models.trello_objects import TrelloCard, TrelloList
+
+
+class Status(Enum):
+    ToDo = "ToDo"
+    Completed = "Completed"
+
+
+class Item:
+    status: Status
+
+    def __init__(self, id, title, status: Status = Status.ToDo):
+        self.id = id
+        self.title = title
+        self.status = status
+
+    @classmethod
+    def from_trello_card(cls, card: TrelloCard, list: TrelloList):
+        return cls(card.id, card.name, Status[list.name])
```

Comment:
Minor, but worth noting that I don't think this plays ball with any statuses that might have spaces in, since it's looking at the enum keys. We might instead use `Status(list.name)` to work the other way round which feels nicer to me for converting potentially user-facing-text into the enum

Reply:
Unsure why but I have a built in fear of using Enums by the value rather than the key... however I cannot justify that fear in any way and I agree with your logic so will update 👍